documentation/Camara_Versioning_Guidelines.md

```diff
@@ -0,0 +1,28 @@
+# Camara subproject release guidelines
```

Comment:
Missing `versioning-pic.png` in images directory.

Reply:
added few mins ago